lib/Crate.js

```diff
@@ -5,7 +5,8 @@ var os = require('os'),
   mmm = require('mmmagic'),
   Magic = mmm.Magic,
   check = require('check-types'),
-  FileProcessor = require('./FileProcessor')
+  FileProcessor = require('./FileProcessor'),
+  targetBuilder = require('./targetBuilder')
```

Comment:
My bad, missed that in refactoring. In the first pass, targetBuilder was a constructor.